packages/shared/src/hooks/useUserMention.ts

```diff
@@ -31,7 +31,7 @@ import {
 import { nextTick } from '../lib/func';
 import { useRequestProtocol } from './useRequestProtocol';
 
-interface UseUserMention {
+export interface UseUserMention {
```

Comment:
I'm sorry I meant can't we just export this as `UseUserMentionProps` it's a bit vague now with two times the same export `useUserMention` and `UseUserMention`

Reply:
we already have a type as `UseUserMentionProps` it is the input types `useUserMention` function takes. this is the output.

```

function useUserMention(props: UseUserMentionProps): UseUserMention { ... }

```

Reply:
Renamed to `UseUserMentionOptions`